platform/android/MapboxGLAndroidSDK/.gitignore

```diff
@@ -0,0 +1,2 @@
+lint-baseline.xml
+lint/lint-baseline-local.xml
```

Comment:
@zugaldia at the moment, if you run this locally and remove any error, you'll get a warning from the command line advising you to remove it from the baseline. If you removed it you should remove it too from the CI baseline (which is the only one included in the repo). Eventually, it'll be removed (when we remove all current lint errors included).